lib/matplotlib/tests/test_backend_qt.py

```diff
@@ -114,6 +114,53 @@ def test_fig_close(backend):
     assert init_figs == Gcf.figs
 
 
+@pytest.mark.backend('Qt5Agg')
+def test_fig_close_ioff(qt_module):
```

Comment:
Rename the test perhaps?  Right now the name seems unrelated.

Reply:
Good call, renamed.